src/types/ChampionSpell.ts

```diff
@@ -18,8 +18,8 @@ export interface SpellData {
   cost: number[];
   costBurn: string;
   datavalues: {};
-  effect: (null | number[])[];
-  effectBurn: (null | string)[];
+  effect: (number[] | undefined)[];
```

Comment:
All of the types will have `null` where they are, these take in the type of data that might be returned in the static files and needs to be able to handle all possible values.

Reply:
Same here i guess